packages/mobile/src/components/comments/CommentForm.tsx

```diff
@@ -16,63 +16,80 @@ type CommentFormValues = {
 type CommentFormProps = {
   onSubmit: (commentMessage: string) => void
   initialValue?: string
-  hideAvatar?: boolean
   isLoading?: boolean
   TextInputComponent?: typeof RNTextInput
 }
 
 const messages = {
-  beFirstComment: 'Be the first to comment'
+  beFirstComment: 'Be the first to comment',
+  addComment: 'Add a comment'
+}
+
+type CommentFormContentProps = Omit<
+  CommentFormProps,
+  'onSubmit' | 'initialValue'
+>
+
+const CommentFormContent = (props: CommentFormContentProps) => {
+  const { isLoading, TextInputComponent } = props
+  const { currentUserId, comments } = useCurrentCommentSection()
+  const { submitForm } = useFormikContext()
+
+  const message = comments?.length
+    ? messages.addComment
+    : messages.beFirstComment
+
+  return (
+    <Flex direction='row' gap='m' alignItems='center'>
+      {currentUserId ? (
+        <ProfilePicture
+          userId={currentUserId}
+          style={{ width: 40, height: 40, flexShrink: 0 }}
+        />
+      ) : null}
+      <Box flex={1}>
```

Comment:
nit: should be able to just put this directly onto the TextField via `css` prop instead of needing a wrapper

```tsx
<HarmonyTextField
  css={{flex: 1}}
/>
```

Reply:
nice, that's better 🙏